_posts/2020-11-09-the-cloud-data-warehouse.md

```diff
@@ -0,0 +1,158 @@
+* Lack of high availability - RedShift clusters are limited to operating in a single availability zone. Whilst a cluster can generally recover simply from individual data node failures, a power loss to a whole AZ would require customers to create a new cluster from snapshots in a different AZ
+
+
+At the end of the day, it's what your organisation is capable of supporting that is more important, which is why I also rate this article on the [Two Philosophies of Cost in Data Analytics](https://www.holistics.io/blog/the-two-philosophies-of-cost-in-data-engineering/), discussing the trade off between service cost and labour cost.
+
+
+## Where does this leave specialists of existing data warehouses?
+
+Many important questions arise when people consider migrating to Cloud Data Warehouses, such as
+
+* How to transition skilled database administrators and operations staff to cloud?
+* What extra skills are required with a Cloud Data Warehouses?
+* Is a Cloud Data Warehouse going to lower our TCO for Data Warehousing?
+
+These are complicated questions to answer but it's worth bearing in mind that the effort required to maintain business services doesn't go away, it just changes in nature. As we'll explore in the next article, the core skills of ETL developers are absolutely transferable.
+
+Some new themes emerge with Cloud Data Warehouses; though the technology or implementations may change, the concerns and risk controls will typically be very familiar to experienced data warehouse operations professionals.
+
+### FinOps
+
+[FinOps](https://www.finops.org/what-is-finops/) is concerned with prudent financial management in the Cloud. This is a major sea change for most large organisations that are used to centralising management and procurement of IT hardware. The Cloud Native way is to bring accountability back to teams both for their spend and management of the resources they provision.
+
+To make FinOps successful you need strong, centralised compliance and guardrail frameworks in place to ensure common baseline behaviours and expectations, such as tagging resources with cost codes, enacting soft limits based on policy, etc.
+
+In the case of Snowflake, as an example, the following questions emerge that FinOps should set out to address:
+* How do I configure and align cost accountability for users, teams and projects across our organisation?
+* How do I prevent overspend at a team, project or organisation level?
+* How can I forecast my credit usage and plan capacity purchase?
+* How can I rank each Virtual Warehouse in terms of its efficiency to identify wastage?
+
+### SecOps
+
+[SecOps](https://www.sumologic.com/glossary/secops/) applies to all cloud estates including the domain of Cloud Data Warehouses, where the attack vectors for data leakage, pollution and security compromise differ from traditional perimeter secured, on premise data warehouses. Examples of challenges organisations face in this realm are
```

Comment:
This too is an important section that needs more detail.
It's a sanitary factor. 
You need to address the latent fear that putting my data in the cloud will expose me. 
Highlight the problems with 'security theatre at the firewall' that fools people they have everything udner control.
Use a nice big  example of a data breach from a traditional setting.

Reply:
(if you just frame this bit as 'everything security in cloud is new, and you need to learn' you will just scare people off.